docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -214,6 +179,41 @@ You can use LogQL to wrap a log query with functions that create metrics from yo
 
 For more information about metric queries, refer to the [Loki metric queries documentation](/docs/loki/latest/logql/metric_queries/).
 
+## Review toolbar features
+
+In addition to the **Run query** button and mode switcher, Builder mode provides additional elements:
+
+| Name                      | Description                                                                               |
+| ------------------------- | ----------------------------------------------------------------------------------------- |
+| **Kick start your query** | A list of useful operation patterns you can use to add multiple operations to your query. |
+| **Label browser**         | Used to navigate through your labels and values, and also build queries.                  |
+| **Explain query**         | Displays a step-by-step explanation of all query components and operations.               |
```

Comment:
We have `Explain query` content in `Builder mode` section. As `Explain query` is available in both modes, it would probably make sense to move content under `## Review toolbar features` - bellow `Label browser`